src/include/common/types/dtime_t.h

```diff
@@ -39,6 +39,8 @@ class Time {
 public:
     // Convert a string in the format "hh:mm:ss" to a time object
     KUZU_API static dtime_t FromCString(const char* buf, uint64_t len);
+    KUZU_API static bool TryConvertInterval(
```

Comment:
Can you use camel case for all function/var names?

Reply:
Should I do it for all functions in ```class Time```? I see we've had this format before https://github.com/kuzudb/kuzu/pull/2932/commits/0343c26367c16ba46fb1783a54ec08c0c138a559#diff-1404cac3353b35101b4538a24f6b56427763f368063cf4abee8387f4bdf8ec88L42